### DISCUSSION TITLE
running as podman container with non-root user on ipv6 vm

### DISCUSSION THREAD

**Discussion by ravi5802** [containers/podman#19115] [Q&A]
1.we have created the user called xyz and started the container as xyz user.
2. not able to access the ipv6 google dns values inside the container.
executed curl -kv -g -6 https://google.com/ command.
*** Could not resolve host: google.com

Closing connection 0
curl: (6) Could not resolve host: google.com**
3. /etc/resolv.conf file is looks like below.
====================
search dns.podman
nameserver 172.18.0.1
nameserver fd00:d311:e3c:5c6::1
nameserver 2001:4860:4860::8888
nameserver 2001:4860:4860::8844

is there any option to make it work for non root user.

**Comment by rhatdan** [comment]
@Luap99 PTAL
